content/introduction/concepts.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Concepts"
+description: "Learn more about the core concepts and domain model that make up the SweetOps methodology."
+weight: 3
+---
+
+SweetOps is built on top of a number of high-level concepts and terminology that are critical to understand prior to getting started. In this document, we break down these concepts to help you get a leg up prior to going through your first tutorial.
+
+## Layers
+![4 Layers of Infrastructure](https://lucid.app/publicSegments/view/dc705e05-cf3e-4e03-9029-acd8c4b4812f/image.png)
+
+SweetOps breaks down cloud infrastructure into 4 discrete layers:
+
+1. **Foundational**: Your AWS Accounts, VPCs, IAM, and DNS architecture.
+1. **Platform**: Your EKS / ECS Clusters, Load Balancers, IAM Service Accounts, and Certificates.
+1. **Shared Services**: Your CI/CD pipelines, BeyondCorp solution, and Monitoring and Logging tooling.
+1. **Application**: Your Backend and Frontend Applications.
+
+We delineate between these layers as they have different lifecylces and tools that are responsible for managing them (e.g. Terraform is great for building your Foundational layer, but is not a good solution to manage your Application layer). It's also important to note that as you proceed down the layers, each layer is more likely to change over time: You won't frequently change your AWS VPCs in your Foundational layer or rebuild EKS clusters in your Platform layer, but you might add HashiCorp Vault to your Shared Services to increase your security posture or add a new API microservice to your Application layer.
```

Comment:
```suggestion
We delineate between these layers as they have different Softwar Development Life Cycles (SDLC) and tools that are responsible for managing them differ. For example, Terraform is great for building your Foundational layer, but other tools might be better for managing the continuous delivery of the Application layer (e.g. ArgoCD). It's also important to note that each layer builds on the previous. The lower layers are less likely to change over time. At the bottom, you won't frequently change your AWS accounts and VPCs in your Foundational layer, just like when operating a platform you won't be rebuilding EKS clusters in your Platform layer without disrupting all tenants of the platform. You might, however, add HashiCorp Vault to your Shared Services layer to increase your security posture or add a new API microservice to your Application layer. The Application Layer is ultimately the most important layer of them all: it's what drives your business. It's where your applications live, which is why it will change continuously.
```